Documentation/coding-guidelines/clr-code-guide.md

```diff
@@ -8,7 +8,85 @@ Written in 2006, by:
 - Mike Stall ([@mikestall](https://github.com/mikestall))
 - Rudi Martin ([@Rudi-Martin](https://github.com/Rudi-Martin))
 
-# Why you must read this document
+# Contents
+
+* [1 Why you must read this document](#1)
+  * [1.1 Rules of the Code](#1.1)
+  * [1.2 How do I &lt;insert common task&gt;?](#1.2)
+* [2 Rules of the Code (Unmanaged)](#2)
```

Comment:
@jasonwilliams200OK I think that Roman numerals are usually used to number items of a list, not to number chapters or sections (at least in technical books). Also the style I used is also used in other parts of the documentation.